java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -183,4 +183,23 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Long> zcard(String key);
+
+    /**
+     * Returns the scores associated with the specified <code>members</code> in the sorted set stored
+     * at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zmscore/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param members An array of members whose scores are to be retrieved.
+     * @return An <code>array</code> representing the scores for the specified <code>members</code>.
+     *     <br>
+     *     If a <code>member</code> does not exist, the corresponding value in the <code>array</code>
```

Comment:
```suggestion
     *     If a <code>member</code> does not exist, a <code>null</code> value will be returned in the <code>Array</code>. 
```

Reply:
I like my wording a bit better here I'll keep it. I'll update Array.